apps/www/pages/politikfragebogen/frage/[questionSlug].js

```diff
@@ -0,0 +1,90 @@
+import { createGetServerSideProps } from '../../../lib/apollo/helpers'
+import Page from '../../../components/PoliticsQuestionnaire/SingleQuestionView'
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { csvParse } from 'd3-dsv'
+import { nest } from 'd3-collection'
+import {
+  QUESTION_SEPARATOR,
+  QUESTION_TYPES,
+  QUESTIONS,
+} from '../../../components/PoliticsQuestionnaire/config'
+
+import {
+  leftJoin,
+  getTypeBySlug,
+} from '../../../../www/components/PoliticsQuestionnaire/utils'
+
+export default ({
+  chartAnswers,
+  question,
+  nestedResponses,
+  questionTypes,
+  questionIndex,
+}) => (
+  <Page
+    chartAnswers={chartAnswers}
+    question={question}
+    nestedResponses={nestedResponses}
+    questionTypes={questionTypes}
+    questionIndex={questionIndex}
+  />
+)
+
+export const getServerSideProps = createGetServerSideProps(
+  async ({
+    ctx: {
+      params: { questionSlug },
+    },
+  }) => {
+    const data = await fs.readFile(
+      path.join(
+        process.cwd(),
```

Comment:
Same as in the other two pages, I'd suggest loading it with the PUBLIC_BASE_URL prefix